### PR TITLE
The plot extension allows passing kwargs to bokeh function call.

### PR DESCRIPTION
**Pull request by janchorowski**
Useful to e.g. get a semilogy plot

### CODE REVIEW

File: blocks/extras/extensions/plot.py [hunk 1]
```diff
@@ -44,12 +44,16 @@ class Plot(SimpleExtension):
     document : str
         The name of the Bokeh document. Use a different name for each
         experiment if you are storing your plots.
-    channels : list of lists of strings
+    channels : list of lists of strings or dicts
```
Comment (rizar) [inline blocks/extras/extensions/plot.py]: This doesn't read. I had to read code to understand that it means `list of list of strings OR list of dicts`.

**Comment by janchorowski** [comment]
Updated

**Comment by rizar** [comment]
Please satisfy Scrutinizer and I will press the button.

**Comment by janchorowski** [comment]
Sorry, I relied on GH's web based editor to handle training whitespace.

**Pull request merged**